consensus/ConsensusTableUpdater.py

```diff
@@ -0,0 +1,338 @@
+
+        if variant_classifications[lab] == empty:
+            return ',"{}","{}"'.format(empty, empty), False
+        else:
+            classification = class_map[variant_classifications[lab]]
+            lab_id = lab.upper().replace('_', '')
+            variant_lab_id = lab_id + '_' + chromosome + '_' + start + '_' + ref + '_' + alt + '_' + gene
+            return ',"{}","{}"'.format(variant_lab_id, classification), True
+
+    @staticmethod
+    def get_match_count_if_consensus(matches, classification):
+        """
+        Returns the number of labs that classified the variant if they reached consensus
+        :param matches: number of labs that classified the variant
+        :param classification: the consensus classification
+        :return: if consensus classification: the number of labs that agree upon it, else an empty column
+        """
+        # Consensus can be (Likely) benign/(Likely) pathogenic/VUS and one lab always agrees with itself
+        if '(Likely)' in classification or classification == 'VUS' or classification == 'Classified by one lab':
+            return ',"{}"'.format(str(matches))
+        else:
+            return ',""'
+
+    def get_matching_history(self, variant):
+        """
+        Get the history for the selected variant
+        :param variant: de id of the variant to retrieve history from
+        :return: a list of ids with history of the variant
+        """
+        variant_history = []
+        for export_id in self.history:
+            export = self.history[export_id]
+            history_id = export_id + '_' + variant
+            if history_id in export:
+                variant_history.append(history_id)
+            if history_id + '_dup0' in export:
+                variant_history.append(history_id + '_dup0')
+            if history_id + '_dup1' in export:
+                variant_history.append(history_id + '_dup1')
+        return variant_history
+
+    def delete_comments(self):
+        self._delete_data(self.comments_table, 'Deleting old comments')
+
+    def delete_consensus(self):
+        self._delete_data(self.consensus_table, 'Deleting old consensus')
+
+    def _delete_data(self, table, msg='Deleting data'):
+        """
+        Deletes all data from a table using the molgenis client
+        :param msg: message showing when busy deleting table
+        :param table: fully qualified name of table to delete data from
+        :return: None
+        """
+        with yaspin(text=msg, color='green') as spinner:
+            try:
+                self.molgenis_server.delete(table)
+                spinner.ok('✔')
+            except Exception as e:
+                spinner.fail('💥')
+                cprint("Error while deleting data from [{}]: {}".format(table, e), 'red', attrs=['bold'],
+                       file=sys.stderr)
+
+    def _upload_file(self, file_name):
```

Comment:
Does the same as the upload method in the reporter. Extract to separate class and reuse